apps/frontend/core-dapp/src/stores/graphs/CoreGraphStore.ts

```diff
@@ -7,18 +7,28 @@ import { RootStore as CoreRootStore } from '../../../generated/mst-gql/core-dapp
 import { PositionCostBasis } from '../../types/user.types'
 import { HistoricalEvents } from '../../features/history/history.types'
 
+export type HistoryEventFilter = {
```

Comment:
for consistency purpose could we rename this to `HistoricalEventsFilter`
```suggestion
export type HistoricalEventsFilter = {
```